src/lib/seq2HLA.ml

```diff
@@ -0,0 +1,28 @@
+
+open Common
+open Run_environment
+module K = KEDSL
+
+let hla_type ?host ~work_dir ~run_with ~r1 ~r2 ~run_name =
```

Comment:
The "configuration" consists of hardcoded allele files is handled by biopam; there isn't any. Feel free to factor that out from our own [seq2HLA fork](https://bitbucket.org/hammerlab/seq2hla) and then just update the seq2HLA opam package [info](https://github.com/solvuu/biopam/tree/master/packages/seq2HLA/seq2HLA.2.2)